src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +15,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        User userByEmail = userService.findByEmail(email);
+
+        if (userByEmail == null) {
+            return false;
+        } else {
+            return userByEmail.getPassword().equals(password);
+        }
```

Comment:
```suggestion
        } 
        return userByEmail.getPassword().equals(password);
```
do we need `else` after `return`?